### PR TITLE
updated the readme file on test usage

### PR DESCRIPTION
**Pull request by AnnieJovitha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,25 @@ docker run -p 8080:8080 philipssoftware/bom-base:latest
...
 Unit tests are executed by the Maven `mvn clean test` command.
-
+Test requires scancode-toolkit to be installed.
+
+find how to install and configure based on OS at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install 
+
+On Linux/Mac:
+1. Open a terminal
+2. cd to the clone directory
+3. run ./configure
```
Comment (timovandeput) [inline README.md:113]: I think I am missing this file in this pull-request.

(Please use extension `.sh` to clarify it is a shell script.)

File: README.md [hunk 2]
```diff
@@ -103,7 +103,25 @@ docker run -p 8080:8080 philipssoftware/bom-base:latest
...
+4. run source bin/activate 
+5. create symbolic links in /usr/local/bin to the build results.
+   Now run the tests.
+   
+On Windows:
+1. open a command prompt
+2. cd to the clone directory
+3. run configure
```
Comment (timovandeput) [inline README.md:121]: I think I am missing this batch file in the pull-request.

File: README.md [hunk 3]
```diff
@@ -103,7 +103,25 @@ docker run -p 8080:8080 philipssoftware/bom-base:latest
 ## How to test the software
 
 Unit tests are executed by the Maven `mvn clean test` command.
-
+Test requires scancode-toolkit to be installed.
+
+find how to install and configure based on OS at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install 
+
+On Linux/Mac:
+1. Open a terminal
```
Comment (timovandeput) [inline README.md:111]: Developers would know this step (and the one below), so I don't think it is necessary to mention it here.

**Review by timovandeput — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AnnieJovitha) [reply README.md:113]: i just copy pasted what was in the scancode webpage

File: README.md [hunk 4]
```diff
@@ -58,14 +58,32 @@ The backend is built by the Maven `mvn clean install` command, and yields a
...
 
 Some useful command line parameters are:
 
 - `--server.port=9090` changes the http port to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
 
+### Scancode  
+Install scancode-toolkit to use bombase with scancode option.
+Find how to install and configure based on OS at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install
```
Comment (timovandeput) [inline README.md:71]: Markdown allows you to create a more beautiful link like this: `Follow these [installation instructions](https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install).`

File: README.md [hunk 5]
```diff
@@ -58,14 +58,32 @@ The backend is built by the Maven `mvn clean install` command, and yields a
...
+using `java -jar BOM-base-<version>.jar`.
 
 Some useful command line parameters are:
 
 - `--server.port=9090` changes the http port to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
 
+### Scancode  
+Install scancode-toolkit to use bombase with scancode option.
```
Comment (timovandeput) [inline README.md:70]: ScanCode is not an _option_; it is an integral part of the application.

File: README.md [hunk 6]
```diff
@@ -58,14 +58,32 @@ The backend is built by the Maven `mvn clean install` command, and yields a
...
-using `java -jar BOM-bar-<version>.jar`.
+using `java -jar BOM-base-<version>.jar`.
 
 Some useful command line parameters are:
 
 - `--server.port=9090` changes the http port to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
 
+### Scancode
```
Comment (timovandeput) [inline README.md:69]: I *think* it is spelled as "ScanCode".

File: README.md [hunk 7]
```diff
@@ -104,6 +122,9 @@ docker run -p 8080:8080 philipssoftware/bom-base:latest
 
 Unit tests are executed by the Maven `mvn clean test` command.
 
+Pre-requisite: scancode-toolkit(Refer to the installation section).
```
Comment (timovandeput) [inline README.md:125]: I think it is called "ScanCode Toolkit".

**Review by timovandeput — CHANGES_REQUESTED** [review]

**Review by timovandeput — APPROVED** [review]

**Pull request merged**